lib/rules/indent.js

```diff
@@ -1084,16 +1084,17 @@ module.exports = {
             },
 
             ArrowFunctionExpression(node) {
-                const firstToken = sourceCode.getFirstToken(node);
+                const maybeOpeningParen = sourceCode.getFirstToken(node, node.async ? 1 : 0);
```

Comment:
I've read the docs: 

> countOptions is an object which has 3 properties; count, includeComments, and filter. Default is {count: 0, includeComments: false, filter: null}.

Not sure a number(0/1) is allowed(maybe the docs need to be updated). or just using the "official" options:

```js
const maybeOpeningParen = sourceCode.getFirstToken(node, { count: node.async ? 1 : 0});
```

Reply:
For `getFirstToken` it's `skipOptions`.

Short form for number [is supported](https://github.com/eslint/eslint/blob/488d1595aef43c4d52cccdb2c97977884f0375a8/lib/source-code/token-store/index.js#L249) and I think it's common in our codebase, but it may be indeed more readable to use the long `{ skip: node.async ? 1 : 0 }`.

Reply:
well, seems the [docs](https://eslint.org/docs/developer-guide/working-with-rules#contextgetsourcecode-1) did not mention short form.

Reply:
Yeah, it would be nice to get that documented! On the other hand, that part of the documentation is already pretty difficult to understand as it is...

Reply:
That being said, I like the readability of the long form :)

Reply:
done